examples/CIB/ex0/example.cpp

```diff
@@ -105,10 +106,7 @@ bool
 run_example(int argc, char* argv[])
 {
     // Initialize PETSc, MPI, and SAMRAI.
-    PetscInitialize(&argc, &argv, NULL, NULL);
-    SAMRAI_MPI::setCommunicator(PETSC_COMM_WORLD);
-    SAMRAI_MPI::setCallAbortInSerialInsteadOfExit();
-    SAMRAIManager::startup();
+    IBTKInit::initialize(argc, argv, MPI_COMM_WORLD, NULL, NULL);
     SAMRAIManager::setMaxNumberPatchDataEntries(2054);
```

Comment:
I don't know where 2054 came from, but I think that we should be able to get away with 2048. Or, make the default number higher --- 4096 --- for safety. But, in any case, it would be nice to get rid of these calls to `SAMRAIManager::setMaxNumberPatchDataEntries`.